### PR TITLE
Adding Github Actions

### LINKED ISSUE #15: Setup Github Actions
Setting up GitHub actions would allow us to ensure that code remains in a runnable state always.

### PR DESCRIPTION
**Pull request by jsilvaigor**
Adding initial github actions and also a badge to show the current status on the Readme file. This fixes #15

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm test
```
Comment (YashKumarVerma) [inline .github/workflows/node.js.yml:28]: the package.json doesn not contain these commands, why are they here :cold_sweat: 
```JSON
"scripts": {
    "test": "jest",
    "start:dev": "nodemon src/app.js"
  },
```

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (jsilvaigor) [reply .github/workflows/node.js.yml:28]: Hi @YashKumarVerma, the command `npm ci` is the same as `npm install` but optimized for running in continuous integration environments and the command `npm test` is just a shortcut to `npm run test` that will execute the jest test suite on the application. 
But I've found a bug in the badge, it was markdown inside an html block. I'll update the PR with this fix.

**Review by YashKumarVerma — APPROVED** [review]
LGTM

**Comment by lopeselio** [comment]
PR ready to merge

**Pull request merged**